Add sort bug to known issues (#269)
AY2425S1-CS2103T-F14A-3/tp: docs/UserGuide.md

1. **When using multiple screens**, if you move the application to a secondary screen, and later switch to using only the primary screen, the GUI will open off-screen. The remedy is to delete the `preferences.json` file created by the application before running the application again.
2. **If you minimize the Help Window** and then run the `help` command (or use the `Help` menu, or the keyboard shortcut `F1`) again, the original Help Window will remain minimized, and no new Help Window will appear. The remedy is to manually restore the minimized Help Window.

> 3. **Sorting by Same Name/Deadline:** When a sort is applied, if multiple people share the same name or deadline, modifying other fields (e.g., project status) may shuffle their order, though sorting among different names or deadlines remains correct. This bug is due to a JavaFX SortedList issue and will be fixed in an upcoming update.

Reviewer: ```suggestion
3. **Sorting by Same Name/Deadline:** When a sort is applied, if multiple people share the same name or deadline, modifying other fields (e.g. project status) may shuffle their order, though sorting among different names or deadlines remains correct. This bug is due to a JavaFX SortedList issue and will be fixed in an upcoming update.
```

> 3. **Sorting by Same Name/Deadline:** When a sort is applied, if multiple people share the same name or deadline, modifying other fields (e.g. project status) may shuffle their order, though sorting among different names or deadlines remains correct. This bug is due to a JavaFX SortedList issue and will be fixed in an upcoming update. More details in the [developer's guide](DeveloperGuide.md#appendix-planned-enhancements)

Reviewer 2: ```suggestion
3. **Sorting by Same Name/Deadline:** When a sort is applied, if multiple people share the same name or deadline, modifying other fields (e.g. project status) may shuffle their order, though sorting among different names or deadlines remains correct. This bug is due to a JavaFX SortedList issue and will be fixed in an upcoming update. More details in the [Developer's Guide](DeveloperGuide.md#appendix-planned-enhancements).
```

> 3. **When sorting people with same name/deadline**: In rare cases where multiple people have the exact same name/deadline and you commit a sort operation on that field, modifying one of those clients' other fields (i.e their project status) might cause those with that same name/deadline to be shuffled. The list will still be correctly sorted, but it might be jarring to see. This bug will be addressed soon in an upcoming iteration.

Reviewer: ```suggestion
3. **When sorting people with same name/deadline**: In rare cases where multiple people have the exact same name/deadline and you perform a sort operation based on that field, modifying one of those clients' other fields (i.e their project status) might cause those with that same name/deadline to be shuffled. The list will still be correctly sorted among clients with different name/deadlines, but it might be jarring to see. This bug is an issue with javafx's implementation of `SortedList` and will be addressed soon in an upcoming iteration.
```

> 3. **When sorting people with same name/deadline**: In rare cases where multiple people have the exact same name/deadline and you commit a sort operation on that field, modifying one of those clients' other fields (i.e their project status) might cause those with that same name/deadline to be shuffled. The list will still be correctly sorted, but it might be jarring to see. This bug will be addressed soon in an upcoming iteration.

Reviewer 2: We could maybe add a link to the future enhancements section of the DG?

Author: Hmm, could be a good idea
